base/shared/src/main/scala/scalaz/typeclass/Show.scala

```diff
@@ -1,5 +1,22 @@
 package scalaz
+package typeclass
 
-abstract class Show[A] {
+/** A typeclass describing types which can be meaningfully represented as a `String`.
+  */
+trait Show[A] {
   def show(a: A): String
```

Comment:
I really prefer the `Cord` approach of scalaz7. Consider the case of building up the Show for a nested ADT. And indeed if we leave the `Cord` method abstract it means we can use SAM types when defining instances.

Reply:
I do as well, but `Cord` isn't in scalaz8 yet, right? I'll put a comment saying that once it's in this should be changed.